File-Formats/PDF/Convert-HTML-To-PDF/aspnet-mvc.md

```diff
@@ -49,15 +50,18 @@ Add a new action method ExportToPDF in HomeController.cs and include the below c
 
 // Initialize HTML to PDF converter.
 HtmlToPdfConverter htmlConverter = new HtmlToPdfConverter();
```

Comment:
Continue number for button creation and code

Reply:
added number for button creation and code